en/simulation/README.md

```diff
@@ -213,7 +213,29 @@ A remote computer can then connect to the simulator by listening to the appropri
 
 ### Use MAVLink Router
 
-The [mavlink-router](https://github.com/intel/mavlink-router) can be used to route  packets from localhost to an external interface.
+The [mavlink-router](https://github.com/intel/mavlink-router) can be used to route packets from localhost to an external interface.
+
+To route packets between SITL running on one computer (sending MAVLink traffic to localhost on UDP port 14550), and QGC running on another computer (e.g. at address `10.73.41.30`) you could:
+
+- Start *mavlink-router* with the following command: 
+  ```
+  mavlink-routerd -e 10.73.41.30:14550 -e 127.0.0.1:14550
```

Comment:
```suggestion
  mavlink-routerd -e 10.73.41.30:14550 127.0.0.1:14550
```

The `-e` optional argument is for the `Mode = Normal`, and the last (mandatory) argument of the command is for `Mode = Eavesdropping` (see below). You can have multiple "normal" endpoints with multiple `-e` arguments, but I'm not sure if you can have more than one "eavesdropping" endpoint (at least I've never seen a need for that).

Reply:
Thanks @JonasVautherin That fixes my confusion about the examples in the mavlink-router which use the serial port of the mandatory argument. I had decided they had no argument because they were serial ports rather than IP ports, but this makes more sense. 
Is it reasonable to think of the last argument as the "source" and the -e arguments as the routing targets?